src/prefs/SpectrumPrefs.cpp

```diff
@@ -223,11 +223,12 @@ void SpectrumPrefs::PopulateOrExchange(ShuttleGui & S)
                S.Id(ID_FREQUENCY_GAIN).TieNumericTextBox(XXO("High &boost (dB/dec):"),
                mTempSettings.frequencyGain,
                8);
+
+            S.Id(ID_COLOR_SCHEME).TieChoice(XXO("Color Sche&me:"),
```

Comment:
Also for string freeze, the string above must be exactly as the designer specified in UnusedStrings.h.  Cut and paste this string from there (with the comment too, pasted to the line before the line with the string):

'// i18n-hint Scheme refers to a color scheme for spectrogram colors
XC("Sche&me",            "spectrum prefs"),'